Promise/Promise+Extras.swift

```diff
@@ -50,8 +60,14 @@ public enum Promises {
         })
     }
 
-    /// Fulfills or rejects with the first promise that completes
-    /// (as opposed to waiting for all of them, like `.all()` does).
+    /// Fulfills or rejects with the first promise that completes.
```

Comment:
```suggestion
    /// Fulfills or rejects with the first of the input promises that completes.
```
(Still kind of awkward but slightly more accurate. Open to other phrasings here!)